components/campaigns/AnnoucementHero.js

```diff
@@ -8,15 +9,14 @@ export default function AnnouncementHero({ className = '', small = false }) {
 
   return (
     <div className={`bg-gray-50 border border-gray-200 py-6  ${className} ${small ? 'mb-0' : 'mb-12'}`}>
-      <h2 className="text-xl md:text-3xl font-bold countdown-text-gradient">AsyncAPI Conference (Nov 16-18)</h2>
-      {!small && 
-        <>
-          <p className=" text-gray-700 md:hidden text-xs  my-2 text-center">Connect and innovate with the AsyncAPI Community.</p>
-          <p className=" text-gray-700 hidden md:block max-w-3xl mx-auto  my-2 text-center">The first day of the conference is dedicated entirely for folks that want to contribute to AsyncAPI Initiative. The next two days of the conference are a pure learning experience, community for the community.</p>
-        </>
-      }
+      <h2 className="text-xl md:text-3xl font-bold countdown-text-gradient">AsyncAPI Conference Day 2 is running!</h2>
+      <div className='px-4 mt-4 mx-auto max-w-7xl'>
+        <YouTube
+          id="mWQtnQIqXTw"
+        />
+      </div>
       <div className="mt-8 pb-2 space-x-2">
-        <Button href="https://zoom.us/webinar/register/8316333610674/WN_pv8NDGIoSg2CnF9qsoptWw" target="_blank" text="Register" />
+        <Button href="https://www.asyncapi.com/slack-invite" target="_blank" text="Comment and ask questions" />
```

Comment:
I'm not a big fan of this copy here, as you can also comment and ask questions through Youtube chat, etc.
Totally no strong opinion anyway so 👍